Assets/HoloToolkit/SpatialSound/Scripts/UAudioManager/ActiveEvent.cs

```diff
@@ -134,15 +134,15 @@ private void SetSourceProperties()
 
             if (audioEvent.Spatialization == SpatialPositioningType.SpatialSound)
             {
-                CreateFlatSpatialRolloffCurve();
                 forEachSource((source) =>
                 {
-                    source.rolloffMode = AudioRolloffMode.Custom;
-                    source.SetCustomCurve(AudioSourceCurveType.CustomRolloff, SpatialRolloff);
                     SpatialSoundSettings.SetRoomSize(source, audioEvent.RoomSize);
-                    SpatialSoundSettings.SetMinGain(source, audioEvent.MinGain);
-                    SpatialSoundSettings.SetMaxGain(source, audioEvent.MaxGain);
-                    SpatialSoundSettings.SetUnityGainDistance(source, audioEvent.UnityGainDistance);
+                    source.rolloffMode = AudioRolloffMode.Custom;
+                    source.maxDistance = audioEvent.MaxDistanceAttenuation3D;
+                    source.SetCustomCurve(AudioSourceCurveType.CustomRolloff, audioEvent.AttenuationCurve);
+                    //source.SetCustomCurve(AudioSourceCurveType.SpatialBlend, audioEvent.SpatialCurve);
+                    //source.SetCustomCurve(AudioSourceCurveType.Spread, audioEvent.SpreadCurve);
+                    //source.SetCustomCurve(AudioSourceCurveType.ReverbZoneMix, audioEvent.ReverbCurve);
```

Comment:
If these are no longer needed, we can remove them outright.

Reply:
Removed.